src/operators.jl

```diff
@@ -74,27 +75,12 @@ Edge 4 => 5
 Edge 5 => 4
 ```
```

Comment:
We need to adjust the documentation of this method - the result is not always a directed graph anymore.